vic/drivers/image/Makefile

```diff
@@ -75,6 +75,7 @@ INCLUDES = -I ${DRIVERPATH}/include \
 
 # Uncomment to include debugging information
 CFLAGS  =  ${INCLUDES} ${NC_CFLAGS}  -ggdb -O0 -Wall -Wextra -std=c99 \
+					 -fopenmp \
```

Comment:
@apcraig - what do we need to do to make this work for the CESM driver. I'm guessing there will need to be some tweaks to get this to work in RASM/CESM.

In [`vic_buildexe.csh`](https://github.com/UW-Hydro/VIC/blob/develop/vic/drivers/cesm/bld/vic.buildexe.csh) we don't include any of these flags so I'm guessing they all come from the CESM build system.